migrate-3-to-4/bottoken.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+function reinject_bottoken() {
+    JOB_NAME=$(kubectl -n "$NAMESPACE" get jobs -o json | jq '.items[0].metadata.name' -r)
```

Comment:
We should add `jq` command check as well here as we had the same for `yq` in `exporter.sh`